layers/+emacs/org/README.org

```diff
@@ -773,31 +784,32 @@ are also available.
-| ~SPC m b b~ | org-babel-execute-buffer                 |
-| ~SPC m b c~ | org-babel-check-src-block                |
-| ~SPC m b d~ | org-babel-demarcate-block                |
-| ~SPC m b e~ | org-babel-execute-maybe                  |
-| ~SPC m b f~ | org-babel-tangle-file                    |
-| ~SPC m b g~ | org-babel-goto-named-src-block           |
-| ~SPC m b i~ | org-babel-lob-ingest                     |
-| ~SPC m b I~ | org-babel-view-src-block-info            |
-| ~SPC m b j~ | org-babel-insert-header-arg              |
-| ~SPC m b l~ | org-babel-load-in-session                |
-| ~SPC m b n~ | org-babel-next-src-block                 |
-| ~SPC m b o~ | org-babel-open-src-block-result          |
-| ~SPC m b p~ | org-babel-previous-src-block             |
-| ~SPC m b r~ | org-babel-goto-named-result              |
-| ~SPC m b s~ | org-babel-execute-subtree                |
-| ~SPC m b t~ | org-babel-tangle                         |
-| ~SPC m b u~ | org-babel-goto-src-block-head            |
-| ~SPC m b v~ | org-babel-expand-src-block               |
-| ~SPC m b x~ | org-babel-do-key-sequence-in-edit-buffer |
-| ~SPC m b z~ | org-babel-switch-to-session              |
-| ~SPC m b Z~ | org-babel-switch-to-session-with-code    |
+| Key binding | Description                                    |
+|-------------+------------------------------------------------|
+| ~SPC m b .~ | Enter Babel Transient State                    |
+| ~SPC m b a~ | org-babel-sha1-hash                            |
+| ~SPC m b b~ | org-babel-execute-buffer                       |
+| ~SPC m b c~ | org-babel-check-src-block                      |
+| ~SPC m b d~ | org-babel-demarcate-block                      |
+| ~SPC m b e~ | org-babel-execute-maybe                        |
+| ~SPC m b f~ | org-babel-tangle-file                          |
+| ~SPC m b g~ | org-babel-goto-named-src-block                 |
+| ~SPC m b i~ | org-babel-lob-ingest                           |
+| ~SPC m b I~ | org-babel-view-src-block-info                  |
+| ~SPC m b j~ | org-babel-insert-header-arg                    |
+| ~SPC m b l~ | org-babel-load-in-session                      |
+| ~SPC m b n~ | org-babel-next-src-block                       |
+| ~SPC m b o~ | org-babel-open-src-block-result                |
+| ~SPC m b p~ | org-babel-previous-src-block                   |
+| ~SPC m b r~ | org-babel-goto-named-result                    |
+| ~SPC m b s~ | org-babel-execute-subtree                      |
+| ~SPC m b t~ | org-babel-tangle                               |
+| ~SPC m b u~ | org-babel-goto-src-block-head                  |
+| ~SPC m b v~ | org-babel-expand-src-block                     |
+| ~SPC m b x~ | org-babel-do-key-sequence-in-edit-buffer       |
+| ~SPC m b z~ | org-babel-switch-to-session                    |
+| ~SPC m b Z~ | org-babel-switch-to-session-with-code          |
+| ~SPC i s~   | insert yasnippet (e.g. source block templates) |
```

Comment:
I believe this binding is from the yasnippets layer---is it documented there?

Reply:
Yes, I was aware that I am repeating things here. But I think that does more good than it does harm. However, I do agree that it is better to point to the keybinding (i.e. the functionality) in that layer. I will fix that...